doc/fuzzing.md

```diff
@@ -0,0 +1,66 @@
+Fuzz-testing Bitcoin Core
```

Comment:
Please remove "Core" references in this doc -- please don't rebrand with Unlimited to make it easier for XT and Classic should they choose to merge this.  And there's no need to constantly repeat the client name throughout the docs...